### PR TITLE
Add Windows GooGet tests.

### PR DESCRIPTION
**Pull request by wrigri**

### CODE REVIEW

File: imagetest/cmd/manager/main.go [hunk 1]
```diff
@@ -84,6 +85,14 @@ var (
 		"ubuntu-pro-1604-lts":     "projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-1604-lts",
 		"ubuntu-pro-1804-lts":     "projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-1804-lts",
 		"ubuntu-pro-2004-lts":     "projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-2004-lts",
+		"windows-2012-r2-core":    "projects/windows-cloud/global/images/family/windows-2012-r2-core",
```
Comment (dorileo) [inline imagetest/cmd/manager/main.go:88]: I'd love to see alphabetically ordered lists/lines :-)

File: imagetest/utils/test_utils.go [hunk 2]
```diff
@@ -238,3 +242,83 @@ func GetInterface(index int) (net.Interface, error) {
...
+		t.Fatalf("couldn't get image from metadata")
+	}
+
+	if strings.Contains(image, "-x86") {
+		t.Skip(skipMsg)
+	}
+}
+
+// PowershellOutput holds stdout, stderr and the exit code from running a powershell command.
+type PowershellOutput struct {
```
Comment (dorileo) [inline imagetest/utils/test_utils.go:266]: ProcessStatus would sound better than PoweshellOutput and would embrace future uses avoiding possible duplications

**Review by dorileo — COMMENTED** [review]
PR looking good, just a minor comment - more a rant though :-).

File: imagetest/cmd/manager/main.go (hunk 1 above)
Comment (wrigri) [reply imagetest/cmd/manager/main.go:88]: I can switch the -core files around on my next PR. Interestingly, the order here is just the order that came out when I did "gcloud images list"

The reason that they come out in that order is that they are ordered by image name and not by family, so
windows-server-2016-dc-core-v20220812
sorts before
windows-server-2016-dc-v20220812

**Pull request merged**